public/app/features/dashboard-scene/scene/workers/createDetectChangesWorker.ts

```diff
@@ -0,0 +1,2 @@
+export const createWorker = () =>
+  new Worker(new URL('./DetectChangesWorker.ts', require('url').pathToFileURL(__filename).toString()));
```

Comment:
how does this work with webpack? does it relace this path? or is done inline somewhow?

Reply:
I simplified this, I hate to not be able to use ESM. It was solved for Jest, by mocking the module.